doc/source/whatsnew/v1.5.0.rst

```diff
@@ -180,8 +180,10 @@ Styler
 
 .. _whatsnew_150.notable_bug_fixes.notable_bug_fix2:
 
-notable_bug_fix2
-^^^^^^^^^^^^^^^^
+Groupby/resample/rolling
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- rolling mean / sum / std / var / skew / kurt now output correct result for window with same values (:issue:`42064`, :issue:`30993`)
```

Comment:
There's an _existing_ `Groupby/resample/rolling` below where this whatsnew note can go.

Also this should just mention `var/std` for this PR. Your other PR's can include the other functions

Reply:
My bad, I didn't notice that.